src/main/java/in/projecteka/consentmanager/clients/model/Provider.java

```diff
@@ -1,7 +1,11 @@
-package in.projecteka.consentmanager.link.discovery.model;
+package in.projecteka.consentmanager.clients.model;
 
 import com.fasterxml.jackson.annotation.JsonAlias;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import in.projecteka.consentmanager.link.discovery.model.Address;
+import in.projecteka.consentmanager.link.discovery.model.Identifier;
+import in.projecteka.consentmanager.link.discovery.model.Telecom;
+import in.projecteka.consentmanager.link.discovery.model.Type;
```

Comment:
Referring classes from discovery at clients package doesn't make sense.

Reply:
these were already there. I just moved the client. Anyway, now moved the dependent types also to client package